deploy/stacks/cognito.py

```diff
@@ -100,13 +100,15 @@ def __init__(
                 domain_prefix=f"{resource_prefix.replace('-', '')}{envname}{self.region.replace('-', '')}{self.account}"
             ),
         )
+        id_token_duration = 120 if with_approval_tests else 60
```

Comment:
The default is 30, should we keep the `with_approval_test` False =30?

Reply:
I thought it was 60 minutes - https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_cognito/UserPoolClientProps.html

Reply:
For some reason in my deployments it was set as 30, all good